addons/mail/static/src/core/common/message.js

```diff
@@ -366,6 +372,16 @@ export class Message extends Component {
         return true;
     }
 
+    async onClickCopy() {
+        const { model, id: threadId } = this.message.thread || {};
+        const { id: messageId } = this.message;
+        const pathname = `/mail/${model}/${threadId}/message/redirect/${messageId}`;
+        await browser.navigator.clipboard.writeText(`${getOrigin()}${pathname}`);
+        this.notification.add(_t("The message link was copied to your clipboard!"), {
+            type: "success",
```

Comment:
Nitpicking but that's not an action really requiring a success theme.
This style is usually reserved for actions that required a high commitment and it gives confidence that this has been processed successfully.
Something neutral like `info` sounds semantically more appropriate.